src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.cs

```diff
@@ -49,9 +49,9 @@ public static CallInstruction Create(MethodInfo info, ParameterInfo[] parameters
                 return GetArrayAccessor(info, argumentCount);
             }
 
+#if !FEATURE_DLG_INVOKE
             return new MethodInfoCallInstruction(info, argumentCount);
-#if FEATURE_DLG_INVOKE
```

Comment:
Created https://github.com/dotnet/corefx/issues/13050 to track reviewing these symbols.